src/IO/Observer/VolumeActions.hpp

```diff
@@ -207,7 +207,7 @@ struct WriteVolumeData {
     Parallel::lock(node_lock);
     std::unordered_map<observers::ArrayComponentId, ExtentsAndTensorVolumeData>
         volume_data{};
-    CmiNodeLock file_lock = nullptr;
+    CmiNodeLock file_lock{};
```

Comment:
Let's `NOLINT` these just in case. `CmiNodeLock` changes type depending on the build. @moxcodes is working on wrapping the node lock with our own type.